### ISSUE TITLE
Opening and closing game options causes input to break

### ISSUE THREAD

**Issue by vaughngx4** [saVantCZ/PS5reborn#34]
Using v2.3.0. Selecting the 3 dots (...) next to the "Play Game" button to open the game options and then closing the game options results in no input working whatsoever. "Alt+TAB" out of Playnite and back fixes the input issue. If I select "Edit Fields" and then press back, input is fine. So it seems to be only when leaving the main game options.

P.S. I am the same person from Reddit.

**Comment by saVantCZ** [comment]
Hm... I cannot reproduce it on my end... are you using any 3rd party software for controllers?

**Comment by vaughngx4** [comment]
I'll run more tests when I'm back home. No, I'm using native DS4 controller support. However, I can reproduce the issue using a keyboard as well.

**Comment by vaughngx4** [comment]
```
24-12 11:48:03.824|DEBUG|WindowFactory:Show dialog window Playnite.FullscreenApp.Windows.GameMenuWindowFactory: d59cf6b2-bb5b-4d7b-b55c-4e77857bea90
24-12 11:48:19.886|DEBUG|WindowFactory:Closing window Playnite.FullscreenApp.Windows.GameMenuWindowFactory: d59cf6b2-bb5b-4d7b-b55c-4e77857bea90, True
```
These are the relevant events in the debug log. GameMenuWindowFactory closing seems to cause the theme to lose focus/lose input. Pressing TAB also works to gain control again. The game menu icon seems to still be focused (appears white with the highlight ring around it) when I have no input. If there's any other information I can provide please let me know.

**Comment by saVantCZ** [comment]
Okay i did reproduce it by pressing option button... but opening game menu via option key on controller or X on keyboard doesnt lose focus. 

I will investigate loosing focus when pressing option button directly next to play button.

**Comment by saVantCZ** [comment]
Okay i have fixed the issue, i think i will release hotfix

**Comment by vaughngx4** [comment]
Thanks for the great work. Love the theme. Going to annoy you with another issue soon LOL.

**Comment by vaughngx4** [comment]
> Thanks for the great work. Love the theme. Going to annoy you with another issue soon LOL.

Ignore this, was an issue on my end.

**Comment by saVantCZ** [comment]
Fixed in 2.3.1, released now.
